web/index.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
```

Comment:
d you think we need the second index.jsp?

Reply:
agree